opentreemap/treemap/css/sass/partials/pages/_admin.scss

```diff
@@ -261,22 +261,22 @@
     .role-table {
         table-layout: fixed;
         tr {
-            height: 50px;
-            border-bottom: 1px solid #EEE;
+            //height: 50px;
+            //border-bottom: 1px solid #EEE;
 
             &.active {
                 background-color: $warning-color;
                 background-color: rgba($warning-color, .75);
             }
             th {
-                background: #DDD;
-                padding: 10px;
+            //    background: #DDD;
+            //    padding: 10px;
                 white-space: nowrap;
                 text-overflow: ellipsis;
                 overflow: hidden;
             }
             td {
-                padding: 10px;
+            //    padding: 10px;
```

Comment:
Why check in these commented out CSS properties?  If they won't be used anymore I think these should be deleted.